config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -186,8 +186,8 @@ periodics:
           - --provider=gce
           # *Manager jobs are skipped because they have corresponding test lanes with the right image
```

Comment:
I think this comment still applies--why remove the skips for the managers? have we removed the manager specific jobs on v1? if so, can you update the comment?

Reply:
I'm not really sure. I wanted to make the jobs equivalent between cgroup v1 and cgroupv2. But I can maybe just increase the timeout rather than change this if you think thats best.

Reply:
yeah let's start with the timeout change, and maybe do a more thorough review of which tests are testing what

Reply:
IMO we shouldn't really have different arguments for a test for cgroup v1 versus cgroup v2. I think it was an oversight with the change.

I think it was a mistake to filter out these jobs for cgroup v1 and not for v2 but I will do the timeout change first and see.

Reply:
@ffromani  do you know if that was intentional or an oversight?

Reply:
I think I added that actually. From what I remember, I noticed that the "manager" jobs all require beefy nodes with more complicated resources. And the jobs are being skipped but still running some of the tests. I exclude them so we can avoid running the jobs. I don't think there was a reason that we targed cgroupv1/cgroupv2.

I think we should try a similar timeout first however.